kubeflow-components/overlays/moc/profiles/demo-profile.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: kubeflow.org/v1beta1
+kind: Profile
+metadata:
+    name: kf-demo
+spec:
+    owner:
+        kind: User
+        name: asanmukh@redhat.com
```

Comment:
we probably want to encrypt this field if it points to an user. Can this be set to a `Group` instead?

Reply:
Good idea :+1: